sqlmesh/core/engine_adapter/bigquery.py

```diff
@@ -475,13 +482,86 @@ def _create_table_properties(
         if clustered_by:
             properties.append(exp.Cluster(expressions=[exp.column(col) for col in clustered_by]))
 
-        for key, value in (table_properties or {}).items():
-            properties.append(exp.Property(this=key, value=value))
+        if table_description:
+            properties.append(
+                self._build_description_property_exp(table_description),
+            )
+
+        properties.extend(self._table_properties_to_expressions(table_properties))
+
+        if properties:
+            return exp.Properties(expressions=properties)
+        return None
+
+    def _build_col_comment_exp(
+        self, col_name: str, column_descriptions: t.Dict[str, str]
+    ) -> t.List[exp.ColumnConstraint]:
+        comment = column_descriptions.get(col_name, None)
+        if comment:
+            return [
+                exp.ColumnConstraint(
+                    kind=exp.Properties(
+                        expressions=[
+                            self._build_description_property_exp(comment),
+                        ]
+                    )
+                )
+            ]
+        return []
+
+    def _build_view_properties_exp(
+        self,
+        table_properties: t.Optional[t.Dict[str, exp.Expression]] = None,
+        table_description: t.Optional[str] = None,
+    ) -> t.Optional[exp.Properties]:
+        """Creates a SQLGlot table properties expression for view"""
+        properties: t.List[exp.Expression] = []
+
+        if table_description:
+            properties.append(
+                self._build_description_property_exp(table_description),
+            )
 
         if properties:
             return exp.Properties(expressions=properties)
         return None
 
+    def _create_comments(
```

Comment:
If I am seeing this correctly, this duplicates the same logic as base but just changes the exec SQL syntax to match what BQ expected. Want to add a `_build_comment_exp` and then have BQ override it what what you are doing here? MySQL/Spark would also be updated to follow this pattern.